pkgs/top-level/all-packages.nix

```
@@ -20060,6 +20060,20 @@ with pkgs;
       inherit darwin;
     });
 
+  qt63 = recurseIntoAttrs (makeOverridable
```

The Qt project no longer makes LTS releases, so there should never be more than one version of Qt 6 in Nixpkgs:

```suggestion
  qt6 = recurseIntoAttrs (makeOverridable
```